workers/unity/Packages/com.improbable.gdk.core/Utility/ViewCommandBuffer.cs

```diff
@@ -37,11 +37,23 @@ public ViewCommandBuffer(EntityManager entityManager, ILogDispatcher logDispatch
                 typeof(Action<Entity, ComponentType, object>), entityManager, setComponentObjectMethodInfo);
         }
 
+        /// <summary>
+        ///     Adds a component to an ECS entity.
```

Comment:
could this be confusing with ECS component? 

maybe call it GameObject component?

Reply:
Yes good idea - can do a `cref` to make it explicit